doc/usage/configuration.rst

```diff
@@ -1765,28 +1765,39 @@ information.
    It must be a list of tuples ``(startdocname, targetname, title, author,
    documentclass, toctree_only)``, where the items are:
 
-   * *startdocname*: document name that is the "root" of the LaTeX file.  All
-     documents referenced by it in TOC trees will be included in the LaTeX file
-     too.  (If you want only one LaTeX file, use your :confval:`master_doc`
-     here.)
-   * *targetname*: file name of the LaTeX file in the output directory.
-   * *title*: LaTeX document title.  Can be empty to use the title of the
-     *startdoc*.  This is inserted as LaTeX markup, so special characters like a
-     backslash or ampersand must be represented by the proper LaTeX commands if
-     they are to be inserted literally.
-   * *author*: Author for the LaTeX document.  The same LaTeX markup caveat as
+   *startdocname*
+     String that specifies the relative path (without file suffix) from the
```

Comment:
I think "document name" is a common term of Sphinx. So using ```:term:`document name` ``` instead is much better for this case.